recipes/gnu-config/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "cci.20200712":
```

Comment:
The last release is too old?

Reply:
GNU config is too cool for doing releases: https://savannah.gnu.org/projects/config

Reply:
It is bundled with `automake` and `libtool` in a subdirectory
But I would like to add it as a package to update these files without having to run `autoreconf`.

Reply:
Okay, but for projects without any releases at all we are using pure date base versioning without the `cci.` prefix

Reply:
I think the latest docs are more clear https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#what-version-should-packages-use-for-libraries-without-official-releases but this does appeare to match the docs

Reply:
My previous comment is outdated. In the meantime we indeed switched to use `cci.` as a prefix for all versions we are creating ourselves.